src/components/home/Contact.js

```diff
@@ -38,98 +71,77 @@ class Contact extends Component {
     this.emailField = ref;
   }
   
-  setSubmitMobileContainerRef(ref) {
-    this.submitMobileContainer = ref;
-  }
-  
-  setDesktopBtnRef(ref) {
-    this.desktopBtn = ref;
+  onSerializing() {
+    if (this.state.school && this.state.email) {
+      this.sendContactInfo();
+    }
   }
   
-  setMobileBtnRef(ref) {
-    this.mobileBtn = ref;
+  onComplete() {
+    this.schoolField.clearInput();
+    this.emailField.clearInput();
+
+    setTimeout(() => {
+      this.setState({
+        status: status.STATIC,
+        school: '',
+        email: ''
+      });
+    }, 1000);
```

Comment:
Nit personally I felt 1 second to be a little short for the user to get feedback that clicking did anything. Can we make it more like 3 seconds? And also extract it to a descriptively-named constant please.